pkg/oc/bootstrap/docker/up.go

```diff
@@ -448,6 +448,9 @@ func (c *CommonStartConfig) Complete(f *osclientcmd.Factory, cmd *cobra.Command,
 		return taskPrinter.ToError(err)
 	}
 	taskPrinter.Success()
+	if len(c.RoutingSuffix) == 0 {
```

Comment:
@deads2k so in EC2 the routing prefix already include the IP?

Reply:
> @deads2k so in EC2 the routing prefix already include the IP?

people have to specify it, but they don't need to take any after launch configuration steps.

Reply:
@mfojtik no, in ec2 the machine doesn't have an interface with its public ip, but that's what's needed for routes to work. So you must specify `[public-ip].nip.io` as the routing suffix.